.github/workflows/ci.yaml

```diff
@@ -60,11 +60,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          # Should we run the tests on macOS too?
-          # - target: "x86_64-apple-darwin"
-          #   os: macos-latest
-          # - target: aarch64-apple-darwin
-          #   os: macos-latest
+          - target: "x86_64-apple-darwin"
+            os: macos-latest
```

Comment:
while we're at it, what about adding arm64 too?

Reply:
Great call! I have considered this too.

Unfortunately, GitHub Actions don't offer GitHub-hosted macOS runners on Apple Silicon yet. This feature is currently planned for Q4 2023 - see https://github.com/github/roadmap/issues/528

I don't think we can run arm64 binaries on x64 machines, can we?

Reply:
Right, makes sense! Let's wait :)